pipelines/pilot_radiosonde/config/dataset.yaml

```diff
@@ -0,0 +1,103 @@
+attrs:
+  title: pilot_radiosonde
+  description: To be updated #to be updated
+  location_id: cali #To be updated 
+  dataset_name: pilot_radiosonde #could change
+  data_level: a1
+
+coords:
+  time:
+    dims: [time]
+    dtype: datetime64[ns]
+    attrs:
+      long_name: UTC time
+      standard_name: time
+      units: Seconds since 1970-01-01 00:00:00 UTC
+
+data_vars:
+  id:
+    dims: []
+    dtype: float
+    attrs: 
+      units: "1"
+  air_temperature:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: air_temperature
+      long_name: Air Temperature
+      units: degC
+  dew_point_temperature:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: dew_point_temperature
+      long_name: Dew Point Temperature
+      units: K
+  air_pressure:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: air_pressure
+      long_name: Air Pressure
+      units: hPa
+  relative_humidity:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: relative_humidity
+      long_name: Relative Humidity
+      units: "%"
+  wind_speed:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: wind_speed
+      long_name: Wind Speed
+      units: m/s
+  wind_to_direction:
+    dims: [time]
+    dtype: float
+    attrs:
+      standard_name: wind_to_direction
+      long_name: Wind Direction (direction wind is coming from)
```

Comment:
If this variable is the direction in the "from" convention, the standard name is "wind_from_direction". Since "from" is typical, you could also rename the variable name to "wind_direction", and remove the parentheses from "long_name".